### ISSUE TITLE
No button is working 

### ISSUE THREAD

**Issue by GaneshKudikala** [mohnishbasha/terraform-ui#1]
I have deployed the application but seems to be no button is working settings,profile,save

**Comment by srinirama** [comment]
yes. for me too . application UI is up . but No button was working
